es73x/src/main/scala/tech/beshu/ror/es/rradmin/rest/RestRRAdminAction.scala

```diff
@@ -27,8 +26,8 @@ import tech.beshu.ror.adminapi._
 import tech.beshu.ror.es.rradmin.{RRAdminAction, RRAdminRequest, RRAdminResponse}
 
 @Inject
-class RestRRAdminAction(settings: Settings, controller: RestController)
-  extends BaseRestHandler(settings) with RestHandler {
+class RestRRAdminAction(controller: RestController)
+  extends BaseRestHandler(null) with RestHandler {
```

Comment:
there's only difference between es73x, and es74x. `BaseRestHandler(null)`, and `BaseRestHandler()`